pages/docs/reference/system-events/inngest-function-failed.mdx

```diff
@@ -0,0 +1,87 @@
+# `inngest/function.failed` {{ className: "not-prose" }}
+
+The `inngest/function.failed` event is triggered whenever any single function fails in your Inngest [environment]((/docs/platform/environments)).
+
+This event can be used to track all function failures in a single place, enabling you to send metrics, alerts, or events to external systems like Datadog or Sentry for all of your Inngest functions.
+
+The TypeScript SDK offers a short hand `onFailure` handler that can be used to handle this event for a specific function.
```

Comment:
Python SDK has an `on_failure` handler. Should we have some sort of language support list for features? Various features have varying SDK support

Reply:
```suggestion
The TypeScript SDK offers a shorthand `onFailure` handler that can be used to handle this event for a specific function.
```

Reply:
I'll make this more generic like "our sdks offer short hand "on failure" handlers to that can be used to handle this for a specific function then link to both in the end of the page.